stable/console-chart/templates/console-deployment.yaml

```diff
@@ -101,8 +100,6 @@ spec:
         - mountPath: /app/certs
           name: {{ template "console.fullname" . }}-console-certs
         image: {{ .Values.global.imageOverrides.console }}
-        command: ["node"]
```

Comment:
Why is this being removed?

Reply:
That was because we were overriding the containers run command. I think it is cleaner to use the containers run command unless there is a reason not to.  I have another gerrit PR to tweak the run command in the container.